pages/osx/applecamerad.md

```diff
@@ -2,6 +2,7 @@
 
 > Camera manager.
 > It should not be invoked manually.
+> More information: <https://www.theiphonewiki.com/wiki/Services>.
```

Comment:
This doesn't look useful.

Reply:
[applecamerad command man page](https://www.reddit.com/r/osx/comments/t7b1xg/applecamerad_command_man_page/) - my question

Reply:
Hard to find something specific.